util/Migrator/DbScripts/2021-03-22_00_Send_Add_HideEmail_Column.sql

```diff
@@ -0,0 +1,154 @@
+-- Add HideEmail column
```

Comment:
I believe you need to update the table file in the SQL project as well

Reply:
& the stored procedure files. Pretty much all SQL objects have dedicated files in the SQL project.

Reply:
Ditto